tests/test_validate_outputs.py

```diff
@@ -90,7 +90,7 @@ def test_calexp(self):
                 ("summary.psfIyy", summary.psfIyy, 4.735538416914092, standard_atol),
                 ("summary.psfIxy", summary.psfIxy, -0.5823368254962096, standard_atol),
                 # TODO: Find a way to tighten psfArea atol in DM-46415.
-                ("summary.psfArea", summary.psfArea, 82.62854203870441, 7e-6),
+                ("summary.psfArea", summary.psfArea, 82.62854203870441, 1e-5),
```

Comment:
@TallJimbo / @erykoff this does fix the test failure but I need pipelines sign off before applying. It fixes this:
```
AssertionError: np.True_ is not false : 82.62855049275065 != 82.62854203870441; diff=8.454046238171031e-06/82.62855049275065=1.0231386352242425e-07 with rtol=0, atol=7e-06
E   summary.psfArea
```

@roceb is going to do a test with the new Google aarch64 chips to see if it passes without this patch.

Reply:
and on new arm chip it also fails without this:
```
SUBFAIL tests/test_validate_outputs.py::TestValidateOutputs::test_calexp - AssertionError: np.True_ is not false : 82.62855049275065 != 82.62854203870441; diff=8.454046238171031e-06/82.62855049275065=1.0231386352242425e-07 with rtol=0, atol=7e-06
summary.psfArea
```
Failure is identical (so that's "good").

Reply:
Change is fine with me.  It's above floating point epsilon but still small enough that I can imagine plenty of plausible explanations (mostly involving one of the input sources to this calculation falling in or out of the sample due to much smaller changes that are thresholded), and it's still not large enough for me to feel compelled to look for more than plausibility.